Ctor 457 new pack network juniper mseries netconf (#4416)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md

> import Tabs from '@theme/Tabs';

Reviewer: Juniper EX Series SNMP

import TabItem from '@theme/TabItem';

## Connector dependencies

The following monitoring connectors will be installed when you install the **Juniper EX Series** connector through the

> **Configuration > Monitoring Connectors** menu:

Reviewer: ```suggestion
**Configuration > Connectors > Monitoring Connectors** menu:
```

> **Configuration > Monitoring Connectors** menu:

Reviewer 2: ```suggestion
**Configuration > Connectors > Monitoring Connectors** menu:
```

| Service Alias  | Service Template                     | Service Description                  |
|:---------------|:-------------------------------------|:-------------------------------------|

> | Cpu-Routing    | Net-Juniper-Ex-Cpu-Routing-custom    | Check CPU Usage of routing engine    |

Reviewer: ```suggestion
| Cpu-Routing    | Net-Juniper-Ex-Cpu-Routing-custom    | Check CPU Usage of routing engine    |
```
```suggestion
| Cpu-Routing    | Net-Juniper-Ex-Cpu-Routing-custom    | Check CPU usage of routing engine    |
```

> | Cpu-Routing    | Net-Juniper-Ex-Cpu-Routing-custom    | Check CPU Usage of routing engine    |

Reviewer 2: ```suggestion
| Cpu-Routing    | Net-Juniper-Ex-Cpu-Routing-custom    | Check CPU usage of routing engine    |
```

| Hardware       | Net-Juniper-Ex-Hardware-custom       | Check hardware                       |

> | Memory-Routing | Net-Juniper-Ex-Memory-Routing-custom | Check Memory Usage of routing engine |

Reviewer: ```suggestion
| Memory-Routing | Net-Juniper-Ex-Memory-Routing-custom | Check memory usage of routing engine |
```

> | Memory-Routing | Net-Juniper-Ex-Memory-Routing-custom | Check Memory Usage of routing engine |

Reviewer 2: ```suggestion
| Memory-Routing | Net-Juniper-Ex-Memory-Routing-custom | Check Memory usage of routing engine |
```

> The services listed above are created automatically when the **Net-Juniper-EX-custom** host template is used.

</TabItem>
<TabItem value="Not attached to a host template" label="Not attached to a host template">

| Service Alias        | Service Template                           | Service Description                                                                                                                     | Discovery  |
|:---------------------|:-------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------|:----------:|

> | Disk-Generic-Id      | Net-Juniper-Ex-Disk-Generic-Id-custom      | Check the rate of free space on the disk (use the ID). Thresholds can be in percentage or in free space remaining                       |            |

Reviewer: ```suggestion
| Disk-Generic-Id      | Net-Juniper-Ex-Disk-Generic-Id-custom      | Check the rate of free space on the disk (use the ID). Thresholds can be in percentage or in free remaining                        space |            |
```

> | Disk-Generic-Id      | Net-Juniper-Ex-Disk-Generic-Id-custom      | Check the rate of free space on the disk (use the ID). Thresholds can be in percentage or in free space remaining                       |            |

Reviewer 2: ```suggestion
| Disk-Generic-Id      | Net-Juniper-Ex-Disk-Generic-Id-custom      | Check the rate of free space on the disk (use the ID). Thresholds can be in percentage or in free remaining space |            |
```

| Disk-Generic-Name    | Net-Juniper-Ex-Disk-Generic-Name-custom    | Check the rate of free space on the disk (use the Name. Difficult to use                                                                |            |

> | Disk-Global          | Net-Juniper-Ex-Disk-Global-custom          | Check the rate of free space on the disk (use the Name. Difficult to use                                                                | X          |

Reviewer: difficult to use?

> | Disk-Global          | Net-Juniper-Ex-Disk-Global-custom          | Check the rate of free space on the disk (use the Name. Difficult to use                                                                | X          |

Reviewer 2: ```suggestion
| Disk-Generic-Name    | Net-Juniper-Ex-Disk-Generic-Name-custom    | Check the rate of free space on the disk (use the Name.                                                                 |            |
| Disk-Global          | Net-Juniper-Ex-Disk-Global-custom          | Check the rate of free space on the disk (use the Name.                                                                 | X          |
```

> | Traffic-Generic-Id   | Net-Juniper-Ex-Traffic-Generic-Id-custom   | Check the bandwidth of the interface. For each checks the name of the interface will appear ("label" shortcut describing the interface) |            |

Reviewer: ```suggestion
| Traffic-Generic-Id   | Net-Juniper-Ex-Traffic-Generic-Id-custom   | Check the bandwidth of the interface. For each check the name of the interface will appear ("label" shortcut describing the interface) |            |
```

> | Traffic-Generic-Name | Net-Juniper-Ex-Traffic-Generic-Name-custom | Check the bandwidth of the interface. For each checks the name of the interface will appear ("label" shortcut describing the interface) |            |

Reviewer: ```suggestion
| Traffic-Generic-Name | Net-Juniper-Ex-Traffic-Generic-Name-custom | Check the bandwidth of the interface. For each check the name of the interface will appear ("label" shortcut describing the interface) |            |
```

> | Traffic-Global       | Net-Juniper-Ex-Traffic-Global-custom       | Check the bandwidth of  interfaces. For each checks the name of the interface will appear ("label" shortcut describing the interface)   | X          |

Reviewer: ```suggestion
| Traffic-Global       | Net-Juniper-Ex-Traffic-Global-custom       | Check the bandwidth of  interfaces. For each check the name of the interface will appear ("label" shortcut describing the interface)   | X          |
```

> | Traffic-Global       | Net-Juniper-Ex-Traffic-Global-custom       | Check the bandwidth of  interfaces. For each checks the name of the interface will appear ("label" shortcut describing the interface)   | X          |

Reviewer 2: ```suggestion
| Traffic-Generic-Id   | Net-Juniper-Ex-Traffic-Generic-Id-custom   | Check the bandwidth of the interface. For each check the name of the interface will appear ("label" shortcut describing the interface) |            |
| Traffic-Generic-Name | Net-Juniper-Ex-Traffic-Generic-Name-custom | Check the bandwidth of the interface. For each check the name of the interface will appear ("label" shortcut describing the interface) |            |
| Traffic-Global       | Net-Juniper-Ex-Traffic-Global-custom       | Check the bandwidth of  interfaces. For each check the name of the interface will appear ("label" shortcut describing the interface)   | X          |
```

### Pack

The installation procedures for monitoring connectors are slightly different depending on [whether your license is offline or online](../getting-started/how-to-guides/connectors-licenses.md).

1. If the platform uses an *online* license, you can skip the package installation
instruction below as it is not required to have the connector displayed within the

> **Configuration > Monitoring Connectors** menu.

Reviewer: ```suggestion
**Configuration > Connectors > Monitoring Connectors** menu.
```

> **Configuration > Monitoring Connectors** menu.

Reviewer 2: ```suggestion
**Configuration > Connectors > Monitoring Connectors** menu.
```

</Tabs>

2. Whatever the license type (*online* or *offline*), install the **Juniper EX Series** connector through

> the **Configuration > Monitoring Connectors** menu.

Reviewer: ```suggestion
the **Configuration > Connectors > Monitoring Connectors** menu.
```

> the **Configuration > Monitoring Connectors** menu.

Reviewer 2: ```suggestion
the **Configuration > Connectors > Monitoring Connectors** menu.
```

| Macro           | Description                                                                                        | Default value     | Mandatory   |
|:----------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER          | Filter operating (default: C\<'routing\|fpc'\>)                                                    | routing           |             |

Reviewer: C?

> | FILTER          | Filter operating (default: C\<'routing\|fpc'\>)                                                    | routing           |             |

Reviewer 2: ```suggestion
| FILTER          | Filter operating                                                   | routing           |             |
```

The expected command output is shown below:

```bash

> OK: All interfaces are ok | 'interface_name1#interface.traffic.in.bitspersecond'=17989b/s;;;; 'interface_name2#interface.traffic.in.bitspersecond'=68335b/s;;;; 'interface_name1#interface.traffic.out.bitspersecond'=46076b/s;;;; 'interface_name2#interface.traffic.out.bitspersecond'=22412b/s;;;; 

Reviewer: thresholds in output

> OK: All interfaces are ok | 'interface_name1#interface.traffic.in.bitspersecond'=17989b/s;;;; 'interface_name2#interface.traffic.in.bitspersecond'=68335b/s;;;; 'interface_name1#interface.traffic.out.bitspersecond'=46076b/s;;;; 'interface_name2#interface.traffic.out.bitspersecond'=22412b/s;;;; 

Reviewer 2: ```suggestion
OK: All interfaces are ok | 'interface_name1#interface.traffic.in.bitspersecond'=19b/s;80;90;; 'interface_name2#interface.traffic.in.bitspersecond'=68b/s;80;90;; 'interface_name1#interface.traffic.out.bitspersecond'=46b/s;80;90;; 'interface_name2#interface.traffic.out.bitspersecond'=22b/s;80;90;; 
```
